index.js

```diff
@@ -0,0 +1,83 @@
+let middleDiv = document.querySelector("#middleDiv");
+let selectList = document.querySelector("#selectList");
+let allMoviesData;
+
+document.addEventListener("DOMContentLoaded", () => {
+    console.log("DOM loaded");
+    fetchMovies();
+    document.getElementById("selectList").addEventListener("change", function() {
```

Comment:
Another way of doing this would be passing "event" argument into that callback function and grab even.target.value inside resultsFromOption function. Both ways work though.